src/assets/voorhoede-core.css

```diff
@@ -1 +1,648 @@
\ No newline at end of file
+.link,
```

Comment:
since this file no longer contains exactly the "core styles" of the voorhoede-ocelot project, maybe it's time to rename / move this file to `src/index.css`?